src/ray/raylet/worker_pool.cc

```diff
@@ -827,7 +831,10 @@ Status WorkerPool::RegisterDriver(const std::shared_ptr<WorkerInterface> &driver
   const auto job_id = driver->GetAssignedJobId();
   HandleJobStarted(job_id, job_config);
 
-  if (driver->GetLanguage() == Language::JAVA) {
+  // TODO do we want to support PrestartWorkers in Julia? Might cut down on runtime overhead...
+  if (driver->GetLanguage() == Language::JAVA || driver->GetLanguage() == Language::JULIA) {
+    send_reply_callback(Status::OK(), port);
+  } else if (driver->GetLanguage() == Language::JULIA) { // TODO check what send_reply_callback does
```

Comment:
these are redundant (will never hit second block for `Language.JULIA`)

```suggestion
  if (driver->GetLanguage() == Language::JAVA) {
    send_reply_callback(Status::OK(), port);
  } else if (driver->GetLanguage() == Language::JULIA) { // TODO check what send_reply_callback does
```

Reply:
thanks - rebased some changes and missed this